packages/fx-core/src/plugins/solution/fx-solution/utils/progressHelper.ts

```diff
@@ -1,10 +1,9 @@
 import { IProgressHandler, PluginContext } from "@microsoft/teamsfx-api";
-import { Messages } from "../constants";
-
-export const DeployArmTemplatesSteps = {
-  DeployArmTemplates: Messages.ProgressDeployArmTemplates,
-};
+import { getStrings } from "../../../../common";
 
+export enum DeployArmTemplatesSteps {
+  ExecuteDeployment = "Deploying solution Arm templates to Azure. This could take several minutes.",
```

Comment:
ARM template

Reply:
This is one thing I want to make consistent. From [Docs](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview), `ARM templates` and `ARM template` are both valid proper noun.

Reply:
I prefer to use `ARM template` since template files are usually multiple.

Reply:
The key point is `Arm` -> `ARM`... I don't have any preference on the plurality. Sorry for the confusing...
For whether use plurality, I saw you're using `ARM templates` in other places, may be you can just align with other strings if there's no special reason to change it. Of curse you can change it to better align with current context.